### ISSUE TITLE
Non optional params on resource update

### ISSUE THREAD

**Issue by mcelicalderon** [twilio/twilio-php#321]
*Note: These issues are for bugs and feature requests for the helper libraries. If you need help or support, please email help@twilio.com and one of our experts will assist you!*

**Version:** 5.0.0-RC6
**API Subdomain (api/taskrouter/ip_messaging):** API

### Code Snippet
```php
public function update($ipAddress, $friendlyName) {
        $data = Values::of(array(
            'IpAddress' => $ipAddress,
            'FriendlyName' => $friendlyName,
        ));
        
        $payload = $this->version->update(
            'POST',
            $this->uri,
            array(),
            $data
        );
        
        return new IpAddressInstance(
            $this->version,
            $payload,
            $this->solution['accountSid'],
            $this->solution['ipAccessControlListSid'],
            $this->solution['sid']
        );
    }
```
That code snippet comes from the next-gen library [here](https://github.com/twilio/twilio-php/blob/next-gen/Twilio/Rest/Api/V2010/Account/Sip/IpAccessControlList/IpAddressContext.php#L69-L89). Most update methods on context objects take an array of optional parameters for the update operation. Here the update method takes two non optional parameter. This won't allow the user to update only one of the two possible parameters.

**Comment by ihumanable** [comment]
Our definition has both of these as `required`, checking with the API implementation.

**Comment by jingming** [comment]
Released with `5.0.0-RC9`.
